### DISCUSSION TITLE
[Support]: Frigate 14 beta 1 runtime errors

### DISCUSSION THREAD

**Discussion by bubba925** [blakeblackshear/frigate#11639] [General Support] [answered]
### Describe the problem you are having

Running the new beta and have a few logs labeled as unknown. Functionality all seems fine and stable but tried searching and couldnt find anyhting about the errors.

### Version

0.14.0-b97e274

### Frigate config file

```removed
```

### Relevant log output

```shell
Unknown
2024-05-28 21:19:40
unknown
/usr/local/lib/python3.9/dist-packages/pandas/core/dtypes/astype.py:133: RuntimeWarning: overflow encountered in cast
Unknown
2024-05-28 21:19:40
unknown
return arr.astype(dtype, copy=True)
Unknown
2024-05-28 21:20:46
unknown
/usr/local/lib/python3.9/dist-packages/pandas/core/dtypes/astype.py:133: RuntimeWarning: overflow encountered in cast
Unknown
2024-05-28 21:20:46
unknown
return arr.astype(dtype, copy=True)
Unknown
2024-05-29 00:40:13
unknown
/usr/local/lib/python3.9/dist-packages/pandas/core/dtypes/astype.py:133: RuntimeWarning: overflow encountered in cast
Unknown
2024-05-29 00:40:13
unknown
return arr.astype(dtype, copy=True)
Unknown
2024-05-29 00:40:14
unknown
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
Unknown
2024-05-29 00:40:14
unknown
/ (part["motion"].max() - part["motion"].min())
Unknown
2024-05-29 00:40:14
unknown
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
Unknown
2024-05-29 00:40:14
unknown
/ (part["motion"].max() - part["motion"].min())
Unknown
2024-05-29 00:40:14
unknown
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
Unknown
2024-05-29 00:40:14
unknown
/ (part["motion"].max() - part["motion"].min())
Unknown
2024-05-29 00:40:14
unknown
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
Unknown
2024-05-29 00:40:14
unknown
/ (part["motion"].max() - part["motion"].min())
Unknown
2024-05-29 00:40:14
unknown
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
Unknown
2024-05-29 00:40:14
unknown
/ (part["motion"].max() - part["motion"].min())
Unknown
2024-05-29 00:40:14
unknown
/opt/frigate/frigate/api/review.py:467: RuntimeWarning: invalid value encountered in scalar subtract
Unknown
2024-05-29 00:40:14
unknown
/ (part["motion"].max() - part["motion"].min())
```

### FFprobe output from your camera

```shell
Dont think needed.
```

### Operating system

Proxmox

### Install method

Docker Compose

### Coral version

PCIe

### Network connection

Wired

### Camera make and model

Doorbell POE reolink, rest dahua

**Comment by hawkeye217** [comment, marked as answer]
These errors are already fixed in the next beta and are only because you were viewing old footage from 0.13 that didn't have any new 0.14 motion data associated with it.

**Reply by bubba925** [reply]
Appreciate it!

**Comment by dolfelt** [comment]
@hawkeye217 I am still getting similar errors in the `0.14.0-beta3` release.

```
/usr/local/lib/python3.9/dist-packages/pandas/core/dtypes/astype.py:133: RuntimeWarning: overflow encountered in cast
return arr.astype(dtype, copy=True)
```

I am assuming it's just from old footage like I've seen mentioned here and elsewhere. It would be nice if the errors could be suppressed and the UI respond accordingly about old footage.

I just started using Frigate a few days ago. Should I just clear out my DB and start over for a smoother experience?

**Reply by hawkeye217** [reply]
I added https://github.com/blakeblackshear/frigate/pull/11795 to display a “no motion data” message in the frontend. This should be part of beta 3. 

Other than the backend log message, are you not seeing this message on the Motion Review pane?

**Reply by dolfelt** [reply]
It seems inconsistent. Now that I have more data on the server, the message is appearing, but every so often if I load too far back it just seems to become unresponsive with some 404s in the console. I'll dig into it more and see if I can provide additional details from the logs.
